packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx

```diff
@@ -1,6 +1,339 @@
 ---
 title: Livestream Tutorial
-description: How to build a livestream experience using Stream's video SDKs
+description: How to build a live stream experience using Stream's React Video SDKs
 ---
 
-TODO
+## Quickstart
+In this quickstart, we'll build a live streaming experience that's similar to Twitch.
+
+- The live stream will run on Stream's Edge network of servers around the world
+- You can use ultra-low latency (WebRTC) based live streaming or HLS (slight delay, but better buffering)
+- We'll show how to publish to the live stream from your browser
+- Stream enables you to scale to millions of viewers
+- You can have multiple active participants in the live stream
+
+The app that we are going to build would work in two modes: Host (presenter) and Viewer (audience).
```

Comment:
```suggestion
The app that we are going to build will work in two modes: Host (presenter) and Viewer (audience).
```

Reply:
It occurred to me we could provide a description about how the app will work, e.g.:

In this app the host will initiate the livestream meanwhile the viewers are waiting for the livestream to start...